unicore/distribute/api/repos.py

```diff
@@ -37,7 +43,14 @@ def collection_post(self):
         repo_name_dot_git = os.path.basename(repo_url_info.path)
         repo_name = repo_name_dot_git.partition('.git')[0]
         try:
-            EG.clone_repo(repo_url, os.path.join(storage_path, repo_name))
+            repo = EG.clone_repo(
+                repo_url, os.path.join(storage_path, repo_name))
+            model_mappings = self.request.validated['models']
+            self.request.registry.notify(
+                RepositoryCloned(
+                    config=self.config,
+                    repo=repo,
+                    mapping=model_mappings))
```

Comment:
Yeah, there's potential for that but I don't think the added complexity at this point is worth it?